addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -65,9 +71,15 @@
                             <span t-if="o.move_type == 'out_invoice' and o.state == 'cancel'">
                                 فاتورة ملغاة
                             </span>
-                            <span t-if="o.move_type == 'out_refund'">
+                            <span t-if="o.move_type == 'out_refund' and o.state == 'posted'">
                                 إشعار خصم
                             </span>
+                            <span t-if="o.move_type == 'out_refund' and o.state == 'draft'">
+                                إشعار خصم المسودة
+                            </span>
+                            <span t-if="o.move_type == 'out_refund' and o.state == 'cancel'">
+                                إشعار خصم ملغاة
+                            </span>
```

Comment:
```suggestion
                            <span t-elif="o.move_type == 'out_refund' and o.state == 'draft'">
                                إشعار خصم المسودة
                            </span>
                            <span t-elif="o.move_type == 'out_refund' and o.state == 'cancel'">
                                إشعار خصم ملغاة
                            </span>
```

Reply:
@epictete Absolutely. Thought I'd done it, but Monday brain ... :smile: There you go!